scripts/init_database.sh

```
@@ -149,39 +146,36 @@ execute_init_sql() {
 # 验证数据库
 verify_database() {
     log_info "验证数据库结构..."
-    
-    # 设置环境变量使用 SQLite
-    export HRMS_ENV=sqlite
-    
+
     # 检查表是否存在
     cd "${PROJECT_ROOT}"
     local tables_output=$("${BUILD_DIR}/sqlexec" -db "${TEMP_DB_NAME}" -sql ".tables" 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)